server/src/main/java/org/opensearch/action/admin/cluster/settings/TransportClusterUpdateSettingsAction.java

```diff
@@ -264,4 +268,26 @@ public ClusterState execute(final ClusterState currentState) {
         );
     }
 
+    private void validateCompatibilityModeSettingRequest(ClusterUpdateSettingsRequest request, ClusterState state) {
```

Comment:
this doesnt seem like the right place to put this validation. Can we add this validation here 

https://github.com/linuxpi/OpenSearch/blob/47f4304f391f9661135e4eb7bf72a246d9974c6b/server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeService.java#L46